### ISSUE TITLE
Tutorial popups in MysteryGrid

### ISSUE THREAD

**Issue by nathan** [nathan/solveme#27]
> - In puzzle 1, a tutorial popup should appear pointing to the tiles in the well at the left. “Drag and drop these tiles into the grid. Each tile can appear only once in a row or column. The tiles in each of the cages need to make the target number using the operation shown.” Maybe a second (perhaps subsequent) popup say points to a cage and says something like “The clue is 20, • for this cage, so you need to find 3 tiles that multiply to 20.” 
> - In puzzle 1, AFTER they’ve filled in all of the tiles, then have a popup appear that points to the submit button that has the same tutorial language in mobiles.
In any puzzle, if the player has gotten to a point where all of the numbers are highlighted red (or would be if the feature were turned on), a popup can appear that points to the menu icon that says “You can start over on this puzzle.”
> - Other popups: 
>     - The first time subtraction appears: “The two numbers can be subtracted in any order, so you may need more clues before you know which option is correct” (show image of 3,5 and 5,3 with a 2- cage)
>     - The first time division appears: “Just like with subtraction, the two numbers can be divided in any order, so you may need more clues before you know which option is correct" (with image)
>     - First time negative appears in a clue: (popup similar to subtraction) in solve overlay, say “Now you can build with negative numbers in clues” - this automatically changes the setting - can also tell users “You can change this feature in the settings menu if you want to limit your building clues to positive integers.” 
>     - First time fraction appears in a clue….
>     - First time after solving a puzzle with variable expression in overlay: “Now you can build puzzles with variables.” 
>     - First no-op puzzle needs instructions: Popup: “The target number is given in this cage, but there’s no operation. That means it could be any operation, and part of the puzzle is figuring out which operation it is.” (By the way, the first puzzle with a no-op would only have one cage with no operation and the rest with operations.). After solving this, the overlay says “Now you can build no-op puzzles of your own” (show image of “show operation” option in build mode).

**Comment by nathan** [comment]
Marking this as wontfix until we have a better idea of what these should look/act like.
